packages/contracts/src/dollar/utils/collectable-dust.sol

```diff
@@ -1,9 +1,9 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.19;
 
-import "@openzeppelin/contracts/utils/address.sol";
-import "@openzeppelin/contracts/utils/structs/enumerable-set.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+// import "@openzeppelin/contracts/utils/address.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+// import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Looks like SafeERC20 might make this obsolete to import. This commit has no lint errors on this file.